app/src/main/java/com/github/amusinamaria/ui/list/MoviesListFragment.kt

```diff
@@ -33,6 +34,8 @@ class MoviesListFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        viewModel = ViewModelProvider(this).get(MoviesListViewModel::class.java)
```

Comment:
you can use "by viewModels()"

Reply:
I tried this:
`val viewModel: MoviesListViewModel by viewModels()`
but it fails with "RuntimeException: Cannot create an instance of class com.github.amusinamaria.viewmodels.MoviesListViewModel"